ping_server.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+ 
+# -q quiet
+# -c number of pings to perform
+FILE="/var/log/ping_server.log"
+ 
+ping -q -c2 opinion.berkeley.edu > /dev/null
```

Comment:
Please use

```
curl https://opinion.berkeley.edu/pcari/en/landing/
```

We've observed that the site at `https://opinion.berkeley.edu/` often runs OK while `https://opinion.berkeley.edu/pcari/en/landing/` is down. Also, we want to test HTTPS specifically--`ping` uses ICMP.